src/main/java/de/hysky/skyblocker/skyblock/item/ItemCooldowns.java

```diff
@@ -80,6 +80,18 @@ private static int getCooldown4Foraging() {
         return baseCooldown - monkeyPetCooldownReduction;
     }
 
+	// Method to handle item cooldowns with optional condition
+	private static void handleItemCooldown(String itemId, int cooldownTime, boolean additionalCondition) {
+		if (!isOnCooldown(itemId) && additionalCondition) {
+			ITEM_COOLDOWNS.put(itemId, new CooldownEntry(cooldownTime));
+		}
+	}
+
+	// Overloaded method for cases without additional conditions
+	private static void handleItemCooldown(String itemId, int cooldownTime) {
+		handleItemCooldown(itemId, cooldownTime, true);
+	}
+
```

Comment:
It doesn't make sense to place these two methods here. Maybe after `onItemInteract`?